### DISCUSSION TITLE
Local Component State Angular Schematics

### DISCUSSION THREAD

**Discussion by vervallsweg** [salesforce/akita#642] [Ideas]
I didn't find any schematics for generating [Local Component State](https://datorama.github.io/akita/docs/angular/local-state/)s.

I did start adding some for `component-name.state.ts` and adding it as a provider. I could also add some for providers with factory and managing them using a service.

Would there be an interest in adding those schematics through a pr? Are there workarounds I missed?

**Comment by NetanelBasal** [comment]
You are welcome to create a PR.
